sql/driver/driver_it_test.go

```diff
@@ -654,23 +652,6 @@ func populateMap(m *hz.Map, count int, keyFn, valueFn func(i int) interface{}) (
 	return entries, nil
 }
 
-func makeDSN(config *hz.Config) string {
-	ll := logger.InfoLevel
-	if it.TraceLoggingEnabled() {
-		ll = logger.TraceLevel
-	}
-	q := url.Values{}
-	q.Add("cluster.name", config.Cluster.Name)
-	q.Add("unisocket", strconv.FormatBool(config.Cluster.Unisocket))
-	q.Add("log", string(ll))
-	return fmt.Sprintf("hz://%s?%s", config.Cluster.Network.Addresses[0], q.Encode())
-}
-
-func mustDB(db *sql.DB, err error) *sql.DB {
-	it.Must(err)
-	return db
-}
-
```

Comment:
Why this change is included in this pr? What is the motivation of the change?

Reply:
It removes dead code we forgot to remove in another merged PR.